### ISSUE TITLE
Postman interceptor missing in win10 app 

### ISSUE THREAD

**Issue by daniele-pintus** [postmanlabs/postman-chrome-interceptor#56]
I just installed the latest version of Chrome and Postman, then I installed the chrome interceptor extension, but it does not appear on postman. there is a "proxy settings" instead.
the interceptor is not working anyway with the embedded chrome extension.

![postman_interceptor](https://cloud.githubusercontent.com/assets/28985036/26501738/b8452cac-423a-11e7-99f1-3d7057c4a74b.PNG)

**Comment by a85** [comment]
Interceptor is only available for the Chrome version right now. We are looking into porting this to the desktop apps soon.

**Comment by cxiaolng** [comment]
@a85 Which version will add the function?
It's a usefully function to web developer

**Comment by bowlej** [comment]
Any news on this?, I have just moved from using the chrome app (a banner on it told me to move to the native app) but i now cant run my old tests i made in the chrome app as they require the interceptor to work.

**Comment by pumanitro** [comment]
Something new ?

**Comment by neramirez** [comment]
a must have!

**Comment by andersondrosa** [comment]
me too! 😭 😞

**Comment by p0pfan** [comment]
When will it add into postman native app

**Comment by Magneticmagnum** [comment]
This is also a must have for my team.  Please have this before the Chrome version is deprecated.

**Comment by TheScree** [comment]
I have to second this, this was an invaluable aspect to Postman. Really frustrating the desktop app for Windows doesn't support this yet.

**Comment by rodrigodata** [comment]
@Magneticmagnum +1

**Comment by lincome** [comment]
+2

**Comment by pszabo1** [comment]
without this feature, the desktop app is not useful for me.

**Comment by lhm7877** [comment]
me too!

**Comment by blackjackyau** [comment]
without this, why should i use Postman?

**Comment by pmjhonwang** [comment]
hurry up !

**Comment by matthewhaworth** [comment]
I moved to the desktop app on a Mac, and have since moved back as this feature is missing.

**Comment by pumanitro** [comment]
From this : http://blog.getpostman.com/2017/03/14/going-native/

> Previously, users of the Postman Chrome app had to download the Postman Interceptor Chrome extension in order to manage cookies and capture requests in the desktop browser.  With the native apps, this functionality is provided within the app itself.

**Comment by joshcomley** [comment]
@pumanitro Yep, this is a very confusing blog post from them, as I am completely unable to get any proxy working that doesn't completely ruin all other normal HTTPS browsing. They provide absolutely no clue on how to use the native app with Fiddler, for example, or how to properly configure the proxy - other than link to some community posts that didn't work for me, anyway.

Personally, the whole point of Postman was to intercept and play with traffic, and it doesn't do that.

**Comment by HoracioEspinosa** [comment]
I can´t open the chrome extension of Postman 👎 :(

**Comment by jsantanders** [comment]
What is the actual status of this? 
still Postman desktop app without Interceptor?

**Comment by the-supercoach** [comment]
If the interceptor is in the desktop app, I can't find it.

**Comment by doumlap** [comment]
+1

**Comment by sid1260** [comment]
+1

**Comment by pszabo1** [comment]
Any news? Now working with deprecated chrome app for over 6 months and I'm getting a little nervous about when these apps aren't working anymore,

**Comment by ghost** [comment]
Here how to use this functionality.

Install "ProxySwitch Omega" on Chrome Browser. At this extension set the next proxy "yourIp:5555" (yourId = IP of your PC).

Go to Postman -> Interceptor. Set port 5555. 

And thats all =)

**Comment by pszabo1** [comment]
@ZhukovGeorge I do not exactly know what you mean. In the "independent" Postman App, there isn't even a Interceptor Button or Menu.

**Comment by ghost** [comment]
@pszabo1 It is placed near the "Sync" button and is called "Capture API requests with Postman".

You are right I had a mistake called this button as Interceptor.

**Comment by tikaro** [comment]
Currently using the Chrome app with Interceptor.  Unless and until the native app has a way to do the same thing, I can't switch.

**Comment by fahadash** [comment]
@tikaro Where did you get the Chrome app? They don't have it on Chrome Web Store anymore.

**Comment by tikaro** [comment]
@fahadash I got it by Googling "Postman Internepctor" and going to:
https://chrome.google.com/webstore/detail/postman-interceptor/aicmkgpgakddgnaphhhpliifpcfhicfo

**Comment by riskers** [comment]
@ZhukovGeorge can you describe  more detail about interceptor cookie?

**Comment by pszabo1** [comment]
still nothing about this issue?
for us, this is/was a main feature of postman.
the app is still not useable for us, we're still using the desktop-app which is deprecated for over a year now.

**Comment by SikoSoft** [comment]
> @pszabo1 It is placed near the "Sync" button and is called "Capture API requests with Postman".
> 
> You are right I had a mistake called this button as Interceptor.

Finding it through here worked for me... Makes me wonder why they don't other updating their guide considering how long people have been getting confused with this.
